app/helpers/user_course_subjects_helper.rb

```diff
@@ -0,0 +1,12 @@
+module UserCourseSubjectsHelper
+  def load_subject course_user
+    @subjects_all = UserCourseSubject.where("course_user_id = ? ", course_user)
```

Comment:
Viết scope cho cái này nhé 

```suggestion
    @subjects_all = UserCourseSubject.where(course_user_id: course_user_id)
```